util/session/sessionmanager_test.go

```diff
@@ -742,43 +778,8 @@ requestedScopes: ["oidc"]`, oidcTestServer.URL),
 		assert.ErrorIs(t, err, common.ErrTokenVerification)
 	})
 
-	t.Run("OIDC provider is Dex, TLS is configured", func(t *testing.T) {
```

Comment:
Why removing this unit test?

It seems like the new unint tests are not validating `assert.ErrorIs(t, err, common.ErrTokenVerification)`

Reply:
The unit test is shifted below in the same file: https://github.com/argoproj/argo-cd/blob/7a4e873e860c83d05354a6a6dfccb3a1df15d5d3/util/session/sessionmanager_test.go#L712-L744

Reply:
I see that the file already have tests for error: "ErrTokenVerification". I don't mean to add more tests.